borrowing_service/utils.py

```diff
@@ -7,7 +7,7 @@
 
 def check_overdue_borrowings():
     today = datetime.now(pytz.utc)
-    tomorrow = today + timedelta(days=1)
+    tomorrow = today + timedelta(days=1) + timedelta(hours=3)
```

Comment:
Here I added additional 3 hours because of timezones it doesn't set tomorrow day precisely (or maybe it's just for me, but let it be)

Reply:
It's because datetime.now set gsm-0)
I have -2 hours, you have -3)